StoreApp/Controllers/StoreController.cs

```diff
@@ -199,6 +200,37 @@ public ActionResult DeleteProduct(ProductViewModel product)
             return RedirectToAction("Index");
         }
 
+        [HttpGet]
+        public ActionResult Buy(int id)
+        {
+            ProductViewModel productVM = webMapper.config.Map<ProductDTO, ProductViewModel>(productService.Get(id));
+
+            if (productVM == null)
+            {
+                TempData["Message"] = "Product don't add to cart";
```

Comment:
It's right to say **"The product wasn't added to the cart"**

Reply:
fixed